src/main/kotlin/com/chloe/kotlinserv/route/GetCountryStatsRoute.kt

```diff
@@ -0,0 +1,109 @@
+package com.chloe.kotlinserv.route
+
+import com.chloe.kotlinserv.http.HttpMethod
+import com.chloe.kotlinserv.http.HttpRequest
+import com.chloe.kotlinserv.http.HttpResponse
+import com.chloe.kotlinserv.http.HttpRoute
+import com.chloe.kotlinserv.model.CountryStats
+import com.google.gson.Gson
+import com.google.inject.Inject
+import com.zaxxer.hikari.HikariDataSource
+
+class GetCountryStatsRoute @Inject constructor(private val dataSource: HikariDataSource) : HttpRoute {
+    override val endpoint = "/countrystats"
+    override val method = HttpMethod.GET
+
+    override val processFunction = { request: HttpRequest ->
+        val json = Gson()
+        val connection = dataSource.connection
+        val groupLocal = request.queryParameters.getValue("groupLocal").firstOrNull()
+        val startDate = request.queryParameters.getValue("startDate").firstOrNull()
+        val endDate = request.queryParameters.getValue("endDate").firstOrNull()
+
+        if (groupLocal == "true") {
+            connection.use {
```

Comment:
we need to get rid of direct clickhouse SQL code here, lets do some refactoring and extract it to GeoDataWriter (rename interface and implementation to GeoDataService)